packages/cli/src/resolve-module.ts

```diff
@@ -0,0 +1,21 @@
+import type { IFileSystem } from '@file-services/types';
+import { createRequestResolver } from '@file-services/resolve';
+import type { StylableConfig } from '@stylable/core';
+
+export function createDefaultResolveModule(fs: IFileSystem): StylableConfig['resolveModule'] {
+    const moduleResolver = createRequestResolver({ fs });
+
+    return (context, request) => {
+        const { resolvedFile } = moduleResolver(context, request);
+
+        if (resolvedFile === false) {
+            throw new Error(
+                `Stylable CLI does not support browser field 'false' values. "${request}" resolved to 'false' from "${context}"`
```

Comment:
How is stylable core handling it?

Reply:
nvmind https://github.com/wix/stylable/blob/master/packages/core/src/module-resolver.ts#L38